app/server/src/generator/templates/template8/index.js

```diff
@@ -285,6 +299,9 @@ function template8(values: SanitizedValues) {
       ${values.sections
         .map(section => {
           switch (section) {
+            case 'about':
+              return generator.aboutSection(values.basics, headings.about)
```

Comment:
Same as template 1 –

> Because we want to set the `aboutSection` method in the `Generator` type to optional for now, this will have a type error because Flow won't be sure that the `aboutSection` method exists or not. For now, we can just add a check to make sure it exists before using it to get rid of the type error (we can remove this method check at some point once we have the `aboutSection` implemented in every template)

```suggestion
              return generator.aboutSection ? generator.aboutSection(values.basics, headings.about) : ''
```